atlasdb-impl-shared/src/main/java/com/palantir/atlasdb/keyvalue/partition/util/ConsistentRingRangeRequest.java

```diff
@@ -41,4 +41,9 @@ public boolean equals(Object other) {
         ConsistentRingRangeRequest otherCrrr = (ConsistentRingRangeRequest) other;
         return get().equals(otherCrrr.get());
     }
+
+    @Override
+    public int hashCode() {
+        return 13 + rangeRequest.hashCode();
```

Comment:
shouldn't matter, but I think it'd make more sense in case someone ever had a collection with both ConsistentRingRangeRequest and RangeRequests they would probably expect them to hash different